### ISSUE TITLE
Further update to Startpage 

### ISSUE THREAD

**Issue by danarel** [tycrek/degoogle#99]
I wanted to comment on the other post but it was closed. But when it comes to the Startpage thread, it's worth looking at the research we did at PrivacyTools.io and the questions we asked the CEO about the sale of the company.

https://github.com/privacytoolsIO/privacytools.io/issues/1562

Startpage still retains full control and decision making over the company and it's still headquartered in The Netherlands.

**Comment by tycrek** [comment]
Is there a TL;DR version of that thread? Ton's of information on there.

Unrelated, but in [this comment](https://github.com/privacytoolsIO/privacytools.io/issues/1562#issuecomment-562682109) you mention Wire. Wire is still listed in this guide, should that be of concern?

**Comment by tycrek** [comment]
Adding on to my previous comment, the Waterfox entry in browsers mentions that it is now owned by System1. I'm hesitant on adding it back for now, but if I do I will definitely add a relevant blog post from Startpage regarding this.

**Comment by danarel** [comment]
I can type up a short version. 

Wire is not recommended anymore. Reasoning here:

https://www.thinkprivacy.ch/cutting-the-wire

**Comment by danarel** [comment]
When I have time today I can explain more about both.

**Comment by tycrek** [comment]
That would be great, thanks!

**Comment by danarel** [comment]
Okay, sorry, I got busier than I thought and didn't revisit this until now.
First, disclaimer: I have contracted with Startpage recently. My work with them consisted of meeting them as a consultant to discuss various aspects of the privacy community and to discuss their marketing. I am a communications specialist by profession. I also wrote 4 blog posts for their blog about privacy related topics. I submitted my final blog for them this week and that concluded my current contract with them.

Second, I am the creator of ThinkPrivacy.ch and a team member of PrivacyTools.io.

So both Waterfox and Startpage were in part purchased by Privacy One Group, which is owned by System1. System1 has a long history of ad tracking and non-privacy friendly tactics. They did however start Privacy One Group to create a separate entity to capitalize on the growing privacy market.

Two things are important here.

Waterfox - Waterfox was never a privacy focused browser and never actually claimed to be. Because of the tweaks made by the creator, it was a relativly private browser, but that was never its intention. At PrivacyTools, we don't list Firefox forks because they are usually behind the main browser which is by far the most secure browser on the market (even with its flaws) and can be even more privacy focused with a few add-ons. No point in really using a fork like Waterfox in my opinion. It's merger now with System1 seems to me to be in-line with them trying to branch into privacy more and perhaps launch their own Brave. <- That's not insider information, I literally have no idea, but that is what would make sense from my end.

Startpage - The merger with System1 didn't make a lot of sense to me, except that System1 sees that people are taking privacy more seriously and if they don't figure out how to advertise to them, they will start to lose income. Just like DuckDuckGo is partnered with Microsoft and Yahoo to sell ads, this I believe was a move by System1 to compete.

The way I saw it was, Startpage has everything to lose if they become a non-private search engine because that's their entire business model. Why serve searches from Google is you're just going to be Google anyway? If they are tracking, etc and get caught, their business is over. That's not worth the risk. So from a risk assessment standpoint, I think trusting them to remain private is a good call.

Knowing the CEO of Startpage, he takes privacy incredibly seriously, especially in his personal life. You won't find a social media page by him, etc. That's why he created Startpage to begin with. 

While they didn't handle answering consumer questions as well as I think they should have when they announced the sale, it's also worth noting that all the "whistle-blowing" (bad term since they never hid the sale from anyone, it was posted on their site, they just didn't make a big announcement) was done by a disgruntled ex-employee. Much of the timeline of events and drama she caused didn't actually match up with the reality of the situation. (I can go on here about how she tried to smear me as well, but that's another topic).

When the system admin and myself of PrivacyTools reached out for more information about the sale, the CEO and others from Startpage answered every single question we asked. We were very pleased. What we learned was System1 owned more than 51% of the company now, but that the two owners remained in control of running Startpage. 

That's all I can think of right now,but I am happy to answer more questions if I have the answer.

I think my main point is, we have started treating Startpage as some untrustworthy company because of System1, but still recommend DDG, which is partnered with Microsoft and Yahoo ad networks, which are even less trustworthy than System1, or Qwant, which is for the most part an ad company of it's own trying to find a way to monetize it's own searches. 

What makes Startpage's situation seem so different is that an ex-employee continues to devote all of her time creating a narrative of distrust.
